canonical/netplan.io
# Add docs pages

## Done

- Add `/docs` page
- Add discourse module
- Add search on the `/docs` page
- Remove "Reference" from navigation 
- Make a redirect

## QA

 - Go to https://netplan-io-233.demos.haus/
 - Go to /docs page and check the documentation was added properly 
 - [Brief](https://docs.google.com/document/d/1Ld8Jp6eM1AT9Z0G_ocnIRCpOwkQFgmFKe24AZH5Vw5U/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5354
Fixes https://github.com/canonical-web-and-design/web-squad/issues/5355
